ImplementationGuide/markdown/Datenobjekte/Operations.md

```diff
@@ -254,6 +254,8 @@ Beispiel: Absage eines Termins
 }
 ```
 
+Falls die Aktualisierung eines Termins die Veränderung eines der oben genannten Elemente zur Folge hat, z.B. aufgrund einer zeitlichen Verschiebung des Termins, so kann die Appointment-Ressource erneut unter Beibehaltung der id an die $book-Operation übergeben werden. Das Termin-Repository kann so feststellen, ob der in der Termin in abgeänderter Form verfügbar ist.
```

Comment:
typo: ob der in der der Termin --> ob der Termin

Reply:
Typo direkt per Commit gefixt